### DISCUSSION TITLE
Eror LIME

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#487] [Q&A] [answered]
Halo, saya lagi coba kerjakan case yang concrete prediction. Eror seperti ini bagaimana ya

![image](https://user-images.githubusercontent.com/114885734/207893688-4298faef-c8d3-48f6-bf95-6256903b586d.png)

**Comment by nisa-basalamah** [comment]
Dear @19807311 

Apakah terdapat error message yang muncul ketika menjalankan kode tersebut? Apabila ada mohon untuk disertakan pada screenshot yang dilampirkan untuk dapat dibantu lebih lanjut.

Terima kasih

**Reply by 19807311** [reply]
![image](https://user-images.githubusercontent.com/114885734/208044461-658f4478-c77e-4f57-bc38-3473085b3fdb.png)


Mbak ini erornya

**Comment by nisa-basalamah** [comment, marked as answer]
Dear @19807311 

Untuk memperbaiki error tersebut, Anda dapat mencoba saran pada diskusi berikut: https://github.com/teamalgoritma/community/discussions/486

Terima kasih
